boot/freeldr/freeldr/arch/uefi/uefidisk.c

```diff
@@ -415,7 +415,7 @@ UefiSetupBlockDevices(VOID)
         if (EFI_ERROR(Status) || 
             bio == NULL ||
             bio->Media->BlockSize == 0 ||
-            bio->Media->BlockSize > 2048)
+            bio->Media->BlockSize > 4096)
```

Comment:
@DarkFire01 Do you know why you originally put the max bound at 2048?

Reply:
Me being dumb mostly